share/picongpu/tests/FieldAbsorber/cmakeFlags

```diff
@@ -29,63 +29,67 @@
 #   - start with zero index
 #   - increase by 1, no gaps
 
-# Test that standard Yee solver compiles in 3D and 2D
+# Compile test for standard Yee solver in 3D and setup for runtime field absorber test
 flags[0]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=Yee;-DPARAM_PML_SIZE=10;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2'"
-flags[1]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=Yee;-DPARAM_PML_SIZE=10;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
-# Test that arbitrary-order solver compiles in 3D and 2D
-flags[2]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=ArbitraryOrderFDTD<4>;-DPARAM_CFLFACTOR=1.29;-DPARAM_ABSORBER_SIZE=10;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2'"
-flags[3]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=ArbitraryOrderFDTD<4>;-DPARAM_CFLFACTOR=1.29;-DPARAM_ABSORBER_SIZE=10;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_DIMENSION=DIM2'"
+# Setup for reference simulation of runtime field absorber test
+flags[1]="-DPARAM_OVERWRITES:LIST='-DPARAM_FIELDSOLVER=Yee;-DPARAM_PML_SIZE=10;-DPARAM_PML_KAPPA_MAX=1.0;-DPARAM_PML_ALPHA_MAX=0.2;-DPARAM_GRID_SIZE=660'"
+
+# Compile test for standard Yee solver compiles in 2D
```

Comment:
The following 31 setups are not used and do not follow the small/large style of case 0 and 1.
Should we still keep them?

Reply:
This is a point we need to discuss in the dev meeting for two reasons.
1. Up until now, the fieldAbsorberTest is our compile test for all field solvers. If we do not use these in any other compile test, not all of these are compile tested.
2. Since all of these setups demonstrate reasonable values for the (C/U)PML configuration, I would argue to leave them in. But comment all except the first two which are the ones actually required for the runtime test.

Reply:
Okay - valid point - let's discuss this today in the` dev meeting

Reply:
Result from dev meeting: keep it because we also use the tests for compile tests